### ISSUE TITLE
The form title bar of an MDI child form is shown under the main bar in High DPI

### ISSUE THREAD

**Issue by NadezhdaDX** [dotnet/winforms#9047]
### .NET version

All frameworks (.NET 4.8, .NET 7, .NET 8)

### Did it work in .NET Framework?

No

### Issue description

My name is Nadezhda. I'm a software engineer representing the WinForms team at DevExpress.
We found that a child MDI title bar can be shown under the main bar when MDI forms are shown on a screen with High DPI settings. It's important to resolve this issue because a lot of users use High DPI settings on their monitors.
The current behavior:
![Current](https://user-images.githubusercontent.com/37904867/234592050-228d0196-16ad-4754-bd9e-3a38444c9f04.png)
The expected result is that only the main bar will be displayed as it is shown on the main monitor:
![Expected](https://user-images.githubusercontent.com/37904867/234593046-66e939e0-e512-4579-9a72-24e97591df43.png)

### Steps to reproduce

I attached a project ([MDI_PM2.zip](https://github.com/dotnet/winforms/files/11333460/MDI_PM2.zip)) that demonstrates this behavior using only standard forms.
To reproduce this issue, you need to use two monitors. The first monitor (it should be the main monitor) should have 100 % scaling. The second should have non-default scaling (for example, 175%).
In this case, you can just run the attached project and show its form in the main form. After that, move the form to the second monitor.
[MDI_PM2.zip](https://github.com/dotnet/winforms/files/11333460/MDI_PM2.zip)

**Comment by elachlan** [comment]
@Olina-Zhang can your team please test this?

**Comment by NadezhdaDX** [comment]
Hello,

@Olina-Zhang, I would like to note that the child title bar is shown also when MenuStrip is added in the monitor 175%. I highlighted this area in the following screenshot:
![Title](https://user-images.githubusercontent.com/37904867/234810844-4529f0cf-3847-4f56-b342-893f6922907f.png)

Please note that this area is not visible on the monitor 100%.

**Comment by Olina-Zhang** [comment]
> @Olina-Zhang, I would like to note that the child title bar is shown also when MenuStrip is added in the monitor 175%.

Yes, it seems a scaling issue because the child form is set to maximized. It shouldn't see that child title bar.

**Comment by dreddy-work** [comment]
Unfortunately, this is outside of WinForms and controlled by Windows. We have a trackign issue but unlikely to get a fix from windows. https://microsoft.visualstudio.com/OS/_workitems/edit/32462899 

Duplicate  #6259

**Comment by RussKie** [comment]
> Unfortunately, this is outside of WinForms and controlled by Windows. We have a trackign issue but unlikely to get a fix from windows. [microsoft.visualstudio.com/OS/_workitems/edit/32462899](https://microsoft.visualstudio.com/OS/_workitems/edit/32462899)
> 
> Duplicate #6259

I don't think this is a duplicated of #6259 or the Windows bug. Those are about the styling of the MDI child captions, whereas this issue is about the caption being visible - i.e., the calculation of the MDI child geometry is incorrect.

**Comment by dreddy-work** [comment]
Unless we get those theming and their dimension right and match with parent/owner, i believe anything we do here is going to be temporary fixes.
